tutorials/dagster-serverless-jobs/index.mdx

```diff
@@ -0,0 +1,190 @@
+            item = requests.get(
+                f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json"
+            ).json()
+            results.append(item)
+
+        df = pd.DataFrame(results)
+
+        # recorded metadata can be customized
+        metadata = {
+            "num_records": len(df),
+            "preview": MetadataValue.md(df[["title", "by", "url"]].to_markdown()),
+        }
+
+        return Output(value=df, metadata=metadata)
+
+
+    defs = Definitions(
+        assets=[hackernews_top_story_ids, hackernews_top_stories],
+        resources={
+            "io_manager": S3PickleIOManager(
+                s3_resource=S3Resource(
+                    region_name="fr-par",
+                    endpoint_url="https://s3.fr-par.scw.cloud",
+                    aws_access_key_id=client.access_key,
+                    aws_secret_access_key=client.secret_key,
+                ),
+                s3_bucket=S3_BUCKET_NAME,
+            ),
+        },
+    )
+    ```
+
+## Setting up the environment and creating the image
+
+1. Retrieve the following elements:
+    - your [Serverless SQL Database connection string](/serverless/sql-databases/how-to/connect-to-a-database/#how-to-set-up-credentials)
+    - the [access key and secret key](/identity-and-access-management/iam/how-to/create-api-keys/) for your API key
+    - the name of the [Object Storage bucket](/storage/object/how-to/create-a-bucket/) you created
+2. Export the environment variables below:
+    ```bash
+    export PG_CONN_STRING=<your_serverless_database_connection_string>
+    export SCW_ACCESS_KEY=<your_scaleway_access_key>
+    export SCW_SECRET_KEY=<your_scaleway_secret_key>
+    export S3_BUCKET_NAME=<your_bucket_name>
+    ```
+
+3. Build and push the image to your container registry namespace:
+    ```bash
+    docker build -t rg.fr-par.scw.cloud/<your-namespace>/dagster-scaleway-example:latest . &&
+    docker push rg.fr-par.scw.cloud/<your-namespace>/dagster-scaleway-example:latest
+    ```
+    <Message type="note">
+    You can find your container registry name and endpoint in the [Scaleway console](https://console.scaleway.com/registry/namespaces)
+    </Message>
+4. Create a `dagster.yaml` file in the same folder to store the configuration and replace the container image with the one you pushed:
+
+  ```yml
+  run_launcher:
+    module: dagster_scaleway
+    class: ScalewayServerlessJobRunLauncher
+    config:
+      docker_image: rg.fr-par.scw.cloud/<your-namespace>/dagster-scaleway-example:latest
+      env_vars:
+      - PG_CONN_STRING
+      - SCW_ACCESS_KEY
+      - SCW_SECRET_KEY
+      - S3_BUCKET_NAME
+
+  storage:
+    postgres:
+      postgres_url:
+        # Can be obtained from the Scaleway Console
+        env: PG_CONN_STRING
+  ```
+
+## Executing Serverless Jobs using the Dagster interface
+
+1. Run the command below to launch the Dagster user interface locally:
+    ```bash
+    dagster dev
```

Comment:
This will fail when trying to reproduce because we haven't installed the dagster dependencies locally

IMO, the easiest way to fix this is:
1. Instead of adding the dependencies in the Dockerfile, add them to the `setup.py` `install_requires` generated by `dagster project scaffold`. In the dockerfile, they will be installed when running `pip install .` in the scaffold directory
2. In the tutorial, ask the user do a `pip install .` to install the additional dependencies for dagster (like dagster_postgres)